src/main/java/uk/gov/hmcts/reform/sscs/utils/PartyDetailsUtils.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.hmcts.reform.sscs.utils;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.OtherParty;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.YesNo;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.IndividualDetails;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.OrganisationDetails;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.PartyType;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public final class PartyDetailsUtils {
+
+    private PartyDetailsUtils() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    // TODO check with what is done in SSCS-10321-Create-Hearing-POST-Mapping when it is merged
+    public static IndividualDetails getIndividualDetails(OtherParty party, SscsCaseData sscsCaseData) {
+        // Line added to suppress PMD error, sscsCaseData would be needed to get relatedParties
+        sscsCaseData.getAppeal();
+        if (PartyDetailsUtils.getPartyType(party).equals(PartyType.IND)) {
+            return IndividualDetails.builder()
+                    .firstName(party.getName() == null ? null : party.getName().getFirstName())
+                    .lastName(party.getName() == null ? null :  party.getName().getLastName())
+                    .preferredHearingChannel(HearingUtils.getPartyChannel(party.getHearingSubtype()))
+                    .interpreterLanguage(party.getHearingOptions() == null ? null
+                            : party.getHearingOptions().getLanguages())
+                    .reasonableAdjustments(getReasonableAdjustments(party))
+                    .vulnerableFlag(false)
+                    .vulnerabilityDetails(null)
+                    .hearingChannelEmail(party.getHearingSubtype() == null ? null
+                            : Collections.singletonList(party.getHearingSubtype().getHearingVideoEmail()))
+                    .hearingChannelPhone(party.getHearingSubtype() == null ? null
+                            : Collections.singletonList(party.getHearingSubtype().getHearingTelephoneNumber()))
```

Comment:
These methods could use with using the methods in
https://github.com/hmcts/sscs-hearings-api/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java#L174-L193
However, I do notice this email and phone number are not included in this method, can you add party.getHearingSubtype().getHearingVideoEmail() as the first email and party.getHearingSubtype().getHearingTelephoneNumber() as the first telephone number on the list